src/index.html

```diff
@@ -9,6 +9,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class = "header">
+      <a href="#home" class="logo__link">
+        <img
+          class = "logo__img"
+          src="./images/logo.png"
+          alt="home"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="is-active nav__link">Apple</a>
```

Comment:
It is always better to have links in lowercase (not "#Apple", but "#apple)

Again - for now, it is not critical